vsce/server/src/server.ts

```diff
@@ -558,15 +563,29 @@ CallStack (from HasCallStack):
     connection.console.log(`Found pattern: ${m}`);
 
     // ERROR MESSAGE m: error: <json>
+    console.info();
+    console.info('Compilation errors string:');
+    console.info(compileErrors);
     const errorJson: ReachCompilerErrorJSON = JSON.parse(m[0].substring(7));
 
     //connection.console.log(`Tokens: ` + tokens);
-    const linePos = errorJson.ce_position[0] - 1;
-    const charPos = errorJson.ce_position[1] - 1;
+    let linePos = errorJson.ce_position[0] - 1;
+    let charPos = errorJson.ce_position[1] - 1;
     const suggestions = errorJson.ce_suggestions;
     const actualMessage = errorJson.ce_errorMessage;
     const offendingToken = errorJson.ce_offendingToken;
     const reachCompilerErrorCode = errorJson.ce_errorCode;
+    if (reachCompilerErrorCode === "RE0089") {
```

Comment:
This stack trace parsing is likely useful for more than just this particular error code. This is fine for now, but keep it in mind in case a similar issue arises for similar errors.

Reply:
Will do; thank you for your input!